File: DayThree/Java/TaskOne.java
import java.util.Scanner;
import java.util.Random;
	public class TaskOne{
		public static void main(String[] args){
			Random rand = new Random(); 

			Scanner scanner = new Scanner(System.in);
			int pass = 0;
			int fail = 0;
			int counter = 0;
			
			for(; counter <= 10; counter++){
				int question = rand.nextInt(10);
				int question1 = rand.nextInt(50);
				int expectedAnswer = question + question1;
				System.out.print("Question is: " + question + " + " + question1 + "\t");
				int answer = scanner.nextInt();
			
				if(expectedAnswer == answer){
					pass++;
					System.out.println("correct");
				}
				else{
					fail++;
					System.out.println("incorrect");
				}
			}
			System.out.println(pass);
		

			System.out.println(fail); 
			
		}

	}

			
			
		
		

File: DayThree/JavaScript/TaskOne.js
const prompt = require("prompt-sync")();
let pass = 0;
let fail = 0;
let counter = 0;

for(; counter <= 10; counter++){
	let question = Math.floor(Math.random(10));
	question = prompt(Math.random()*20);
	let question1 = Math.floor(Math.random(20)); 
	question1 = prompt(Math.random()*10);
	let expectedAnswer = question + question1;
	answer = prompt("Question is: " + question + " + " + question1 + "\t");
	if(expectedAnswer == answer){
		pass++;
	}
	else{
		fail++;
	}
}
console.log("The total pass is" + pass);
console.log("The total fails is" + fail);
	

			 
			
		 